src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
@stanlemon That is due to the github repository itself being relatively static.  I.e. People rarely if ever change their github repostitory.

These options I can see people changing depending on how they want to publish a specific release and therefore should probably be CLI based.  I'm not sure how we would do that though as it is currently open to have custom publishers / additional publishers.

@malept Maybe we make each `publisher` also implement a commander and output the current publishers `.help()` after the main `publish` help?